client/overview/modal/reset-account/test/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,7 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Reset Account Modal modal is open when is visible is true 1`] = `
+<div
```

Comment:
I wonder if this test is valid? seems like there is no reset account modal in this html output

Reply:
Good catch! Used a better test in 190a7e145, and it's passing.